File: Week3/Day4/daily-challenge.js
let listTasks=[];
let button= document.getElementById('button');
let div = document.getElementsByClassName("listTasks")[0]
let input=document.getElementById("input")

button.addEventListener('click', addTask)


function addTask(){
    event.preventDefault();
    let text=input.value;
    if (text.length==0){
        alert("Enter Something")
    }
    else{
        listTasks.push(text);
        let i_element = document.createElement('i')
        i_element.classList.add("fa-solid")
        i_element.classList.add("fa-xmark")
        let para = document.createElement('p')
        let checkbox=document.createElement("INPUT")
        checkbox.setAttribute('type','checkbox')
        para.append(checkbox," ", i_element ," ",text);
        div.appendChild(para)
        i_element.addEventListener('click',function(){
            para.remove()
        })
        input.value=""

    }
}

File: Week4/Day3/daily_challenge.py
word=input("Enter a word")
word_dict={}
index_count=0



for i in word:
    if i in word_dict:
        word_dict[i].append(index_count)
    if i not in word_dict:
        word_dict[i]=[]
        word_dict[i].append(index_count)

    index_count+=1

print(word_dict)